src/api/test/unit/project_test.rb

```diff
@@ -122,7 +151,7 @@ def test_flags_inheritance
     package2.store
 
     # the all 'disable' build flag shall overwrite the project setting
-    package2.get_flags('build', ['SLE_11_SP4'], Architecture.all).each do |repo|
+    get_flags(package2, 'build', ['SLE_11_SP4'], Architecture.all).each do |repo|
```

Comment:
same here. Drop all testing of the local function get_flags.

We have a rspec for Flag::SpecifiedFlags and that's what is used.